src/Compilers/Helpers/GlobalAssemblyCacheHelpers/GacFileResolver.cs

```diff
@@ -11,7 +11,7 @@
 namespace Microsoft.CodeAnalysis.Scripting
 {
     /// <summary>
-    /// Extends <see cref="MetadataFileReferenceResolver"/> to enable resolution of assembly
+    /// Extends MetadataFileReferenceResolver to enable resolution of assembly
```

Comment:
@jasonmalinowski it's "by strange design".  The `cref` code doesn't care about `internal` accessibility and hence has an ambiguity between the accessible and inaccessible `MetadataFileReferenceResolver`.